packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -38,7 +38,7 @@ export interface ExpansionHub extends RevHub {
 
     // Device Control
     getBulkInputData(): Promise<BulkInputData>;
-    getADC(): Promise<number>;
+    getADC(channel: number, rawMode: boolean): Promise<number>;
```

Comment:
`rawMode` needs to be documented, `channel` should be an `AnalogChannel` enum.

We may want to go further and not even expose this method publicly, instead exposing different methods for the analog channels, the motor currents, the battery voltage, etc.

I believe the servo current does not work correctly in the current firmware, so we may want to consider not exposing a method for that.

Reply:
I'm not a big fan of the AnalogEnum, mainly because the most common use case would be to check the ports. I do like the idea of separate methods.

Reply:
Created separate methods

Reply:
Let's delete this method, since it's very unclear how it works. I think we should also delete the `rawMode` option from the methods that replace this one.

Reply:
The original plan was to leave getADC for reading the analog inputs. I'm thinking now it would be better to rename it 'getAnalogInput' to match the digital version.